recipes/EOBS/recipe.py

```diff
@@ -0,0 +1,27 @@
+"""Author: Norland Raphael Hagen - 08-03-2021
+Pangeo-Forge recipe for E-OBS data (E-OBS: High-resolution gridded mean/max/min temperature, precipitation and sea level pressure for Europe & Northern Africa)"""  # noqa: E501
+
+
+from pangeo_forge_recipes.patterns import ConcatDim, FilePattern, MergeDim
+from pangeo_forge_recipes.recipes import XarrayZarrRecipe
+
+# Filename Pattern Inputs
+target_chunks = {"lat": 705, "lon": 465}
```

Comment:
What about time? Is the goal here to have the data be contiguous (unchunked) in time?